src/Aspire.Dashboard/Otlp/Storage/TelemetryRepository.cs

```diff
@@ -14,16 +16,6 @@
 
 namespace Aspire.Dashboard.Otlp.Storage;
 
-public sealed class TelemetryOptions
-{
-    public int LogCountLimit { get; init; }
-    public int TraceCountLimit { get; init; }
-    public int MetricsCountLimit { get; init; }
-    public int AttributeCountLimit { get; init; }
-    public int AttributeLengthLimit { get; init; }
-    public int SpanEventCountLimit { get; init; }
-}
-
 public sealed class TelemetryRepository
 {
     internal const string LogCountLimitKey = "DOTNET_DASHBOARD_OTEL_LOG_COUNT_LIMIT";
```

Comment:
Shouldn't these be in a PostConfigure too? Where are these used?

Reply:
These should be removed.